tool/lib/webqa-canonical-generator.js

```diff
@@ -165,43 +186,51 @@ class CanonicalGenerator {
 
 
     _generateExamples(tableName, canonicals, valueSample) {
-        let examples = { npp: { examples: [], candidates: [] } };
+        let examples = {};
+        for (let cat of ['npp', 'avp', 'pvp', 'nni', 'npv', 'apv']) {
```

Comment:
Are you handling the new names at all?

Reply:
no, I plan to switch to the new annotations entirely soon